.github/workflows/unit.yml

```diff
@@ -14,19 +14,13 @@ jobs:
   unittests:
```

Comment:
We should continue to run a `lowest-supported` version with the set of lower-versioned dependencies to ensure that everything continues to work there.